components/tools/OmeroPy/src/omero/util/script_utils.py

```diff
@@ -1234,3 +1230,34 @@ def findROIByImage(roiService, image, namespace):
     for roi in results.rois:
         roiList.append(ROIData(roi))
     return roiList
+
+
+def numpyToImage(plane, minMax, type):
+    """
+    Converts the numpy plane to a PIL Image, converting data type if necessary.
+    @param plane The plane to handle
+    @param minMax the min and the max values for the plane
+    @param type the data type to use for scaling
+    """
+
+    import numpy
```

Comment:
Historically I think we tended to import numpy in a try/except or only in the methods it was needed, so that if it was missing you'd still be able to run all the code that didn't depend on it. But maybe we don't need to do that if it's reliably installed by default now?